source/install/prod-ubuntu-16.04.rst

```diff
@@ -0,0 +1,489 @@
+
+2. Make sure the system is up to date with the most recent security
+   patches.
+
+   ::
+
+       sudo apt update
+       sudo apt upgrade
+
+
+Set up Database Server
+----------------------
+
+1.  For the purposes of this guide we will assume this server has an IP
+    address of ``10.10.10.1``.
+2.  Install PostgreSQL 9.5+::
+
+        sudo apt-get install postgresql postgresql-contrib
+
+3.  PostgreSQL created a user account called ``postgres``. You will need
+    to log into that account with::
+
+        sudo -i -u postgres
+
+4.  You can get a PostgreSQL prompt by typing::
+
+        psql
+
+5.  Create the Mattermost database by typing::
+
+        postgres=# CREATE DATABASE mattermost;
+
+6.  Create the Mattermost user by typing::
+
+        postgres=# CREATE USER mmuser WITH PASSWORD 'mmuser_password';
+
+7.  Grant the user access to the Mattermost database by typing::
+
+        postgres=# GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;
+
+8.  You can exit out of PostgreSQL by typing::
+
+        postgres=# \q
+
+9.  You can exit the postgres account by typing::
+
+        exit
+
+10. Allow Postgres to listen on all assigned IP Addresses
+
+    -  Edit the config file::
+
+           sudo vi /etc/postgresql/9.5/main/postgresql.conf
+
+    -  Uncomment ``listen_addresses`` and change ``localhost`` to ``*``
+
+11. Alter ``pg_hba.conf`` to allow the mattermost server to talk to the
+    postgres database
+
+    -  Edit the config file::
+
+        sudo vi /etc/postgresql/9.5/main/pg_hba.conf
+
+    -  Add the following line to the ``IPv4 local connections``::
+
+        host all all 10.10.10.2/32 md5
+
+12. Reload Postgres database::
+
+        sudo systemctl reload postgresql.service
+
+    Check with ``netstat`` command to see postgresql actually running on given ip and port::
+
+        sudo netstat -anp | grep 5432
+
+    Try restarting the postgresql service if reload won't work::
+
+        sudo systemctl restart postgresql.service
+
+13. Attempt to connect with the new created user to verify everything
+    looks good::
+
+        psql --host=10.10.10.1 --dbname=mattermost --username=mmuser --password
+        mattermost=> \q
+
+
+Set up Mattermost Server
+------------------------
+
+1. For the purposes of this guide we will assume this server has an IP
+   address of ``10.10.10.2``
+2. For the sake of making this guide simple we located the files at
+   ``/home/ubuntu/mattermost``.
+3. We have also elected to run the Mattermost Server as the ``ubuntu``
+   account for simplicity. We recommend setting up and running the
+   service under a ``mattermost`` user account with limited permissions.
+4. Download `any version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_ by typing::
+
+       wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz
+
+   Where ``X.X.X`` is typically the latest Mattermost release version, which is currently ``3.4.0``.
```

Comment:
I have updated the guide to not mention a specific version. The bullet point points to the archive overview and I think that should suffice.